x/concentrated-liquidity/position.go

```diff
@@ -378,6 +378,8 @@ func (k Keeper) mintSharesLockAndUpdate(ctx sdk.Context, concentratedPoolId, pos
 	}
 
 	// Create a coin object to represent the underlying liquidity for the cl position.
+	// Q: what is the impact of this on slashing or anything else. I think truncating liqudiity even slightly might cause the final token amount difference
+	// to end up being quite large. I think the risk of this is offset but superfluid risk factor but would like to confirm.
 	underlyingLiquidityTokenized = sdk.NewCoins(sdk.NewCoin(types.GetConcentratedLockupDenomFromPoolId(concentratedPoolId), position.Liquidity.TruncateInt()))
```

Comment:
Note: would like to confirm with reviewers

Reply:
The only potential issue I can see with this is if we had a 100 percent risk factor, which I am fairly certain we will never do. 

Also, even if we did, it would result in the position getting very slightly under slashed, which I also do not think is innately dangerous.

Good call out though. Maybe we add a note to risk factor pointing to this on why we wouldn't want 1:1 ratio for superfluid minting.